package.json

```diff
@@ -143,27 +144,31 @@
     "@fortawesome/free-solid-svg-icons": "^5.15.4",
     "@fortawesome/react-fontawesome": "^0.1.16",
     "@reduxjs/toolkit": "^1.7.1",
-    "bootstrap": "^4.5.3",
+    "@tanstack/react-table": "^8.20.5",
```

Comment:
Where do we need this? We use "react-table"

Reply:
I concur we use `react-table`. I'm not familiar with this react-table dependency either.